site/docs/features/management.md

```diff
@@ -1,177 +1,8 @@
-* Yearly tags for the last 3 years
-
-Tags are automatically named using a `date/` prefix and a zero-extended underscore based naming scheme.
-For example: `date/2019_09_07_15_50` would be a tag for August 7, 2019 at 3:50pm.
-
-!!! warning
-    AutoTags are automatically deleted once the policy rolls-over. As such, if retention is desired post roll-over, manual tags should be created.
-
-AutoTagging is currently done based on the UTC roll-over of each item.
-
-## Manifest Reorganization
-
-!!! info
-    This service is currently in progress and is not yet included in a released version of Nessie.
-
-Rewrites the manifests associated with a table so that manifest files are organized
-around partitions. This extends on the ideas in the Iceberg [`RewriteManifestsAction`](http://iceberg.apache.org/javadoc/0.11.0/org/apache/iceberg/actions/RewriteManifestsAction.html).
-
-!!! note
-    Manifest reorganization will show up as a commit, like any other table operation.
-
-Key configuration parameters:
-
-| Name                 | Default | Meaning                                               |
-|----------------------|---------|-------------------------------------------------------|
-| effort               | medium  | How much rewriting is allowed to achieve the goals    |
-| target manifest size | 8mb     | What is the target                                    |
-| partition priority   | medium  | How important achieving partition-oriented manifests. |
-
-## Compaction
-
-!!! info
-    This service is currently in progress and is not yet included in a released version of Nessie.
-
-Because operations against table formats are done at the file level, a table can start
-to generate many small files. These small files will slow consumption. As such, Nessie
-can automatically run jobs to compact tables to ensure a consistent level of performance.
-
-| Name                 | Default | Meaning                                                                                                              |
-|----------------------|---------|----------------------------------------------------------------------------------------------------------------------|
-| Maximum Small Files  | 10.0    | Maximum number of small files as a ratio to large files                                                              |
-| Maximum Delete Files | 10.0    | Maximum number of delete tombstones as a ratio to other files before merging the tombstones into a consolidated file |
-| Small File Size      | 100mb   | Size of file before it is considered small                                                                           |
-| Target Rewrite Size  | 256mb   | The target size for splittable units when rewriting data.                                                            |
-
-!!! note
-    Compaction will show up as a commit, like any other table operation.
+    New implementation is a work-in-progress. This section will be updated once it is ready.
```

Comment:
I'm not sure we can claim this PR _fixes_ #2020.

On the other hand, this change would hopefully reduce confusion about Nessie GC in doc, so it might be worth committing it as long as we keep #2020 for future, more detailed doc updates.

@snazy : WDYT?

Reply:
#4991  has a commit that touches this entire file - not sure why it's updated at this time.
I'm very much in favor of _not_ merging this.

Reply:
Recent confusion on the mailing list has triggered this PR.